.github/workflows/release_build.yaml

```diff
@@ -66,12 +77,22 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     needs: [validate, integration-test]
+
     permissions:
+      contents: read
+      id-token: write
```

Comment:
Required for the OIDC login with Fulcio (generates a short lived signing key and certificate)